### PR TITLE
storage.go: Make query tikv for ddl too

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
Call CleanUp cmp for ddl binlog too.
may get the txn status from tikv or skip the binlog.

### Check List 

Tests 

 - Unit test
 - Integration test
 Code changes

 Side effects

Related changes

### CODE REVIEW

**Comment by july2993** [comment]
/run-all-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

**Comment by july2993** [comment]
/run-all-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

**Comment by july2993** [comment]
/run-all-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

**Comment by july2993** [comment]
/run-all-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

File: pump/storage/storage.go [hunk 1]
```diff
@@ -486,33 +486,51 @@ func (a *Append) resolve(startTS int64) bool {
...
+	if status.IsCommitted() {
+		// write the commit binlog myself
+		cbinlog := new(pb.Binlog)
+		cbinlog.Tp = pb.BinlogType_Commit
+		cbinlog.StartTs = pbinlog.StartTs
+		cbinlog.CommitTs = int64(status.CommitTS())
+
+		req := a.writeBinlog(cbinlog)
+		if req.err != nil {
+			log.Error("writeBinlog failed", zap.Error(req.err))
```
Comment (GregoryIan) [inline pump/storage/storage.go]:
```suggestion
			log.Error("write missing commit binlog failed", zap.Error(req.err))
```

File: pump/storage/storage.go [hunk 2]
```diff
@@ -486,33 +486,51 @@ func (a *Append) resolve(startTS int64) bool {
...
+			panic(err)
 		}
 
-		log.Info("known txn is committed or rollback from tikv",
-			zap.Int64("start ts", startTS),
-			zap.Uint64("commit ts", status.CommitTS()))
-		return true
+		err = a.metadata.Put(encodeTSKey(req.ts()), pointer, nil)
+		if err != nil {
+			log.Error("put into metadata failed", zap.Error(req.err))
```
Comment (GregoryIan) [inline pump/storage/storage.go]:
```suggestion
			log.Error("put missing binlog into metadata failed", zap.Stringer("binlog", cbinlog), zap.Error(req.err))
```

**Comment by GregoryIan** [comment]
Rest LGTM

**Comment by july2993** [comment]
/run-all-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

**Comment by GregoryIan** [comment]
/run-all-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

**Comment by GregoryIan** [comment]
/run-all-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

**Comment by july2993** [comment]
/run-all-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

**Pull request merged**